### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by Corbainius** [community/community#62796] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Having tried github for the first time I was impressed and considering moving to a paid account but when I got to deployment to my cPanel server using ssh keys it was infuriating. I gave up, deleted my repository and github has lost a prospective paid customer. I will continue without version control and deploying updates manually. I followed every instruction carefully. The reality is that if i have the login info needed it should not be this difficult to implement. My project is small, I do not need perfect security. I am disappointed and frustrated. Way to kill a guys focus github. 

p.s I resent your little fascist tickbox. I did not search for a similar discussion but I was forced to tick the box anyway. Well done. You suck.

**Comment by samus-aran** [comment]
Hi @Corbainius

Thanks for taking the time to join the GitHub Community.

If you're looking for help for this specific topic, you might want to try asking for help or searching for similar discussions (hint hint). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. 

To help us improve the platform then what you can do is to share your constructive feedback on this Discussion space with the correct tags (include examples), and trust that we are always working to make GitHub as awesome as possible. We can't guarantee that every bit of feedback sees action, but we are paying attention.

We're closing this discussion because it has not had recent activity. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by facundofarias** [comment]
You really should try [DeployHQ](https://www.deployhq.com/) 😄
